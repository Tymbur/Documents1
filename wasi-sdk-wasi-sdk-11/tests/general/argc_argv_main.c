#include <stdio.h>

int main(int argc, char *argv[]) {
    puts("hello from argc argv main!");
    return 0;
}
