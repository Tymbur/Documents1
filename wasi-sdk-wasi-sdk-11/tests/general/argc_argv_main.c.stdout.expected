hello from argc argv main!
